sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -216,3 +207,35 @@ function normalizeResponse<T extends HttpResponseField<any> & { eTag?: string }>
 
   return newResponse;
 }
+
+/**
+ * Translates user-facing field names into their `select` equivalents (these can be
+ * seen in the `KnownEnum5`)
```

Comment:
> KnownEnum5

Can you link to this in the comment?

Reply:
Oooh, I probably can. Will do.

Reply:
I did actually try but I'm not importing KnownEnum5 (it's not used in the code at all so the definition isn't there). Nothing seems to happen that's interesting with ctrl+click, for instance either.

Given that this is an internal function anyways, I'm just going to leave it comment-only for now.